rest_specification/get_generate_value.md

```diff
@@ -0,0 +1,22 @@
+# GET: Generate value
+Get generated values for a field 
+
+### Request
+Generate a value using value generators declared for the 'field' of the given entity version. If 'n' is given, generates that many values.
+```
+GET /data/generate/{entityName}/{version}/{field}?n={n}
+```
+Generate a value using value generators declared for the 'field' of the given entities default version. If 'n' is given, generates that many values.
+```
+GET /data/generate/{entityName}/{field}?n={n}
+```
+---
+
```

Comment:
Added example

On Wed, May 4, 2016 at 12:47 PM, Naveen Malik <notifications@github.com>
wrote:

> In rest_specification/get_generate_value.md
> <https://github.com/lightblue-platform/lightblue-docs-specs/pull/41#discussion_r62093379>
> :
>
> > @@ -0,0 +1,22 @@
> > +# GET: Generate value
> > +Get generated values for a field
> > +
> > +### Request
> > +Generate a value using value generators declared for the 'field' of the given entity version. If 'n' is given, generates that many values.
> > +```
> > +GET /data/generate/{entityName}/{version}/{field}?n={n}
> > +```
> > +Generate a value using value generators declared for the 'field' of the given entities default version. If 'n' is given, generates that many values.
> > +```
> > +GET /data/generate/{entityName}/{field}?n={n}
> > +```
> > +---
> > +
>
> Would be nice to start adding examples.
> Example
>
> *Request*
>
> GET /data/generate/test/path.*.to.field?n=3
>
> *Response*
>
> [5000,5001,5002]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-docs-specs/pull/41/files/e3905eb0b3cf22e7b0f66ba84cbe072fe794cdb5#r62093379>
>